component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1286,6 +1286,9 @@ private boolean getConnection(HttpURLConnection httpConnection, AuthenticationCo
                                 httpResponse);
                     }
                     return true;
+                } else {
+                    log.error("Error while sending SMS: error code is " + httpConnection.getResponseCode()
```

Comment:
What is the reason that we used `:` here? normally we are using `:` to add a parameter next. Here we can see another text.

Reply:
`Error while sending SMS. error code ...`

Reply:
Normally we are printing this type of logs.

Reply:
This  change will make the log inconsistent with line 1307. Change both?

Reply:
No need to change. Just needed to clarify.